docs/environment.yml

```diff
@@ -19,4 +19,4 @@ dependencies:
   - tabulate
   - setuptools-rust>=1.5.2
   - ucx-proc=*=cpu
-    #  - rust>=1.60.0
+  - rust>=1.67.0
```

Comment:
Might be worth adding this constraint to the rust build dependency in our nightly recipe